spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -498,4 +498,87 @@
     end # each
   end # add_folder_name
   # rubocop:enable Metrics/LineLength
+
+  # rubocop:disable Metrics/LineLength
```

Comment:
this is turning off rubocop for individual line length -- which maybe doesn't apply to your tests?    I don't see that you need the disable / enable comments here.  (The above had long strings for key and value which made some lines very long, and my reviewer agreed that it made more sense in the above case to follow the pattern than to reduce the line length.)